sdk/schemaregistry/schema-registry-avro/test/utils/mockedRegistryClient.ts

```diff
@@ -61,11 +61,19 @@ export function createTestRegistry(neverLive = false): SchemaRegistry {
   async function getSchemaProperties(
     schema: SchemaDescription,
     _options?: GetSchemaPropertiesOptions
-  ): Promise<SchemaProperties | undefined> {
-    return mapByContent.get(schema.content);
+  ): Promise<SchemaProperties> {
+    const result = mapByContent.get(schema.content);
+    if (result !== undefined) {
+      return result;
+    }
+    throw new Error(`Schema not found: ${schema.content}`);
   }
 
-  async function getSchema(id: string, _options?: GetSchemaOptions): Promise<Schema | undefined> {
-    return mapById.get(id);
+  async function getSchema(id: string, _options?: GetSchemaOptions): Promise<Schema> {
+    const result = mapById.get(id);
+    if (result !== undefined) {
+      return result;
+    }
+    throw new Error(`Schema ID not found: ${id}`);
```

Comment:
I wonder if inverting the checks would look nicer

```suggestion
    if (!result) {
      throw new Error(`Schema ID not found: ${id}`);
    }
    return result;
```